Test/CoreSDK.Test/Shared/DataContracts/DependencyTelemetryTest.cs

```diff
@@ -38,9 +38,9 @@ public void SerializeWritesNullValuesAsExpectedByEndpoint()
             DependencyTelemetry original = new DependencyTelemetry();
             original.Name = null;
             original.Data = null;
-            original.DependencyTypeName = null;
+            original.Type = null;
             original.Success = null;
-            original.DependencyTypeName = null;
+            original.Type = null;
```

Comment:
thanks. fixed. There is a duplication as there used to be setting of both - `DependencyKind` and `DependencyType`. Now it should only be `Type`